virtual_rainforest/main.py

```diff
@@ -181,6 +138,80 @@ def extract_timing_details(
     return start_time, update_interval, raw_interval, end_time
 
 
+def _get_model_sequence(
+    config: Config, models: dict[str, Any], method: str
+) -> dict[str, Any]:  # FIXME dict[str, Any]-> dict[str, Type[BaseModel]]
+    """Get a tuple of the model execution sequence for a given model stage.
+
+    This function uses the ``priority`` sections in model configurations to establish an
+    execution sequence for a given model method (currently, one of ``init`` or
+    ``update``). For example, the configuration:
+
+    .. code-block:: toml
+        [plants.priorities]
+        init = ['abiotic']
+        update = ['abiotic']
+
+    would set that the ``plants`` model should be both initialised and updated before
+    the ``abiotic`` model.
```

Comment:
I was a bit confused when checking the code until I saw that you used `reversed` in the static order. Now it makes sense.

Having said that, do you think it works better to use `priorities` (model X has higher priority than model Y and therefore should run first) or `dependencies` (model Y depends on model X, so X should run first)? The question is more philosophical than practical - either way works fine. In my mind, you need to define an order because a model depends on another, not because it has a higher priority.